linode/manager
# refactor: Finish Remaining MUI Style Migration with codemod

## Description 📝

Finishes the initial work in the MUI v5 Migration project

> [!important] 
> The end of removing `@mui/styles` is in sight and I wanted to put this up to get us over the finish line. Form my limited testing, I haven't found any regressions and don't think running the codemod on the remaining work is that bad of an idea.
> 
>If we think this it too risky, I can close it and we can continue to break up the remaining work. Just wanted to put this up to keep this effort alive, get opinions on how to complete the remaining work, and to show the team how close we are!


Codemod used:
```sh
bunx @mui/codemod v5.0.0/jss-to-tss-react packages/manager/src/
```
## How to test 🧪

### Verification steps 
- Check Cloud Manager UI in general
- For each component in the diff, find it in Cloud Manager and check for UI regressions

## As an Author I have considered 🤔
